src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,82 @@ title = "Releases"
+To use this feature, run `make e2e using=subctl-verify`.
+Verifications can be now specified using the `SUBCTL_VERIFICATIONS` flag, instead of relying on the default behavior.
+e.g.: `make e2e using=subctl-verify SUBCTL_VERIFICATIONS=connectivity,service-discovery`.
+* kubeconfig handling has been revamped to be consistent across all
+  `subctl` commands and to match `kubectl`’s behaviour.
+  * The single-context commands, `cloud-prepare`, `deploy-broker`, `export`,
+  `join`, `unexport` and `uninstall`, now all support a `--context` argument
+  to specify the kubeconfig context to use. kubeconfig files can be
+  specified using either the `KUBECONFIG` environment variable or the
+  `--kubeconfig` argument; `kubectl` defaults will be applied if
+  configured. If no context is specified, the kubeconfig default context
+  will be used.
+  * Multiple-context commands which operate on all contexts by default,
+  `show` and `gather`, support a `--contexts` argument which can be used
+  to select one or more contexts; they also support the `--context` argument
+  to select a single context.
+  * Multiple-context commands which operate on specific contexts,
+  `benchmark` and `verify`, support a `--context` argument to specify the
+  originating context, and a `--tocontext` argument to specify the target
+  context.
+  * `diagnose` operates on all accessible contexts by default, except
+  `diagnose firewall inter-cluster` and `diagnose firewall nat-traversal`
+  which rely on an originating context specified by `--context` and a
+  remote context specified by `--remotecontext`.
+  * Namespace-based commands such as `export` will use the namespace given
+  using `--namespace` (`-n`), if any, or the current namespace in the
+  selected context, if there is one, rather than the `default`
+  namespace.
+  * These commands also support all connection options supported by
+  `kubectl`, so connections can be configured using command arguments
+  instead of kubeconfigs.
+  * Existing options (`--kubecontext` etc.) are preserved for backwards
+  compatibility, but are deprecated and will be removed in the next
+  release.
+
+### Other changes
+
+* The Flannel CNI is now properly identified during join.
+* A new ServiceExport status condition type named Synced was added that indicates whether or not the ServiceImport
+was successfully synced to the broker.
+* Service Discovery now handles updates to an exported service and updates/deletes the corresponding ServiceImport accordingly.
+* Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
+`clusterset.local`. This prevents unnecessary retries.
+* Cloud cleanup for OpenStack now identifies and deletes failed MachineSets.
+* Privileges of the Route Agent and Gateway pods were reduced as they don’t need to access PersistentVolumeClaims and Secrets.
+Be used to verify if `submariner_router` is pinned to the proper Gateway node.
```

Comment:
```suggestion
Be used to verify if `submariner_router` is pinned to the proper Gateway node.
```
We still have this fragment, not clearly related to either of the adjacent points.